Resources/views/CRUD/base_show.html.twig

```diff
@@ -11,6 +11,14 @@ file that was distributed with this source code.
 
 {% extends base_template %}
 
+{% block title %}
+    {{ "title_show"|trans({'%name%': admin.toString(object)|truncate(15) }, 'SonataAdminBundle') }}
```

Comment:
I just tried to be consistent with the other templates e.g. https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Resources/views/CRUD/base_list.html.twig#L27